docs/UserGuide.md

```diff
@@ -140,15 +143,18 @@ Adds a student to the address book.
 ##### Constraints
 
 * **NAME** must only contain alphanumeric characters and spaces.
-* **SCHEDULE** must be in the format of `DAY_OF_THE_WEEK`-`START_TIME`-`END_TIME`.
+* **SCHEDULE** must be in the format of `DAY_OF_THE_WEEK`-`START_TIME`-`END_TIME` (strictly no space in between).
 * **DAY_OF_THE_WEEK** includes `Monday` `Tuesday` `Wednesday` `Thursday` `Friday` `Saturday` `Sunday`.
 * **START_TIME** and **END_TIME** are represented as `HHmm`.
 * **PHONE_NUMBER** should be 8 digits that starts with 6, 8 or 9.
 * **RATE** is the tuition fee per hour. It must meet the following criteria:
+  * It is a positive numeric value with at most 2 decimal places.
   * Minimum: $0.01 (must be a positive value)
   * Maximum: $1000.00 (two decimal places allowed)
+  > Reason for the maximum value: It is unlikely for an undergraduate tutor to have an hourly rate higher than $1000.00.
 * **PAID_AMOUNT** and **OWED_AMOUNT** must be at least 0 with at most 2 decimal places.
       <i>Example: </i> `12.00`, `0.0` or `7`.
+  * Special case: `-0`, `-0.0`, `-0.00` are not allowed!
```

Comment:
Maybe can add in that this is because it will cause the system to act wierdly

Reply:
Should I create a known issue section for this?

Reply:
Good idea